docs/src/main/sphinx/sql/show-create-function.md

```diff
@@ -2,20 +2,60 @@
 
 ## Synopsis
 
-```text
+```sql
 SHOW CREATE FUNCTION function_name
 ```
 
 ## Description
 
-Show the SQL statement that creates the specified function.
+Shows the SQL statement that created the specified function.
 
 ## Examples
 
-Show the SQL that can be run to create the `meaning_of_life` function:
+List all SQL routines and plugin functions in the `default` schema of the
+`brain_catalog`:
+
+```sql
+SHOW FUNCTIONS FROM brain_catalog.default
+```
 
+Example output:
+
+```text
+Function | Return Type | Argument Types | Function Type | Deterministic | Description
+----------+-------------+----------------+---------------+---------------+-------------
+ answer   | bigint      |                | scalar        | true          |
 ```
-SHOW CREATE FUNCTION example.default.meaning_of_life;
+
+List all functions with a name beginning with `ans`:
+
+```sql
+SHOW FUNCTIONS like 'ans%';
+```
+
+Example output:
+
+```text
+ Function | Return Type | Argument Types | Function Type | Deterministic | Description
+----------+-------------+----------------+---------------+---------------+-------------
+ answer   | bigint      |                | scalar        | true          |
+ ```
+
+The following example uses `SHOW CREATE FUNCTION` to reveal the SQL statement
+that created the `answer` function:
```

Comment:
Test this some more with a routine that has a description and such .. also reword .. it shows the `definition of the routine` not the statement used to create it